src/workloads/query/TimeseriesTSBSOptimizations.yml

```diff
@@ -0,0 +1,283 @@
+                  [
+                    {
+                      $match:
+                        { "time": { $lt: { ^Date: "2016-01-18T01:49:23" } } },
+                    },
+                    {
+                      $project:
+                        {
+                          _id: 1,
+                          tags: 1,
+                          usage_softirq: 1,
+                          usage_steal: 1,
+                          usage_guest: 1,
+                        },
+                    },
+                  ]
+
+  # These queries test bucket-targeting, and we want to target a small percentage of buckets.
+  # The task below will target ~300 buckets or .1%.
+  - Name: match_on_measurement
+    Type: CrudActor
+    Threads: 8
+    Phases:
+      OnlyActiveInPhases:
+        Active: [7]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: 100
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationName: aggregate
+              OperationCommand:
+                Pipeline:
+                  # All usage_* fields are random integers between 0-100.
+                  # Verify before adding to this workload how many buckets each query targets.
+                  [
+                    { $match: { usage_guest: 88, usage_nice: 50 } },
+                    {
+                      $project:
+                        {
+                          _id: 1,
+                          tags: 1,
+                          usage_softirq: 1,
+                          usage_steal: 1,
+                          usage_guest: 1,
+                        },
+                    },
+                  ]
+
+  # The task below will target ~580 buckets or .3%.
+  - Name: match_on_measurement_meta
+    Type: CrudActor
+    Threads: 8
+    Phases:
+      OnlyActiveInPhases:
+        Active: [8]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: 100
+          Database: *database
+          Collection: *collection
+          Operations:
+            - OperationName: aggregate
+              OperationCommand:
+                Pipeline:
+                  # All usage_* fields are random integers between 0-100.
+                  [
+                    {
+                      $match:
+                        {
+                          $and:
+                            [
+                              { usage_steal: { $gte: 5, $lt: 20 } },
+                              { "tags.hostname": "host_44" },
+                            ],
+                        },
+                    },
+                    {
+                      $project:
+                        {
+                          _id: 1,
+                          tags: 1,
+                          usage_softirq: 1,
+                          usage_steal: 1,
+                          usage_guest: 1,
+                        },
+                    },
+                  ]
```

Comment:
Don't these need a branch_name?

Reply:
It looks like the "AutoRun" workloads get named after their yml file by transforming Capitalization to underscore. That is,  "FooBar.yml" would map to "foo_bar" task. To make the analysis of the perf results easier, maybe, let's name this file "TimeseriesTsbsOptimizations.yml" and name the corresponding task in DSI "timeseries_tsbs_optimizations"?

Reply:
BTW, connecting the workload definitions to the results in Evergreen might be an argument to create separate DSI tasks for each workload (using Genny's AutoRun naming scheme). Otherwise, one has to search in DSI repo for the mentioning of this *.yml file to see task name which then can be searched in Evergreen... (I do hope the proposal about tagging workloads with "Context" would help with stuff like this).

Reply:
done! Confirmed with Jim offline and split the configurations between TsbsOptimizations workload and TsbsQuery workload. Note that the names of the dsi configurations are slightly different than the genny workloads to match the dsi configuration name style, which is different than the genny workload name style. 

TsbsOptimizations.yml dsi configuration is tsbs_query_optimizations
TsbsQuery.yml dsi configuration is tsbs_query_genny